### PR TITLE
spanish translation

### PR DESCRIPTION
**Pull request by fernandaruizm**
spanish translation

### CODE REVIEW

**Comment by artlowel** [comment]
Thanks @fernandaruizm!

You have some tests that fail right now, but I think merging in, or rebasing on the latest master branch fill fix those.

File: resources/i18n/es.json5 [hunk 1]
```diff
@@ -0,0 +1,1610 @@
...
+  "menu.section.find_private_items": "Ítemes Privados",
+//  "menu.section.find_withdrawn_items": "Withdrawn Items",
+  "menu.section.find_withdrawn_items": "Ítemes Retirados",
+
+//  "menu.section.icon.access_control": "Access Control menu section",
+  "menu.section.icon.access_control": "Sección del menú de Control de Acceso",
+//  "menu.section.icon.control_panel": "Control Panel menu section",
+  "menu.section.icon.control_panel": "Sección del menú de Panel Control",
+//  "menu.section.icon.curation_task": "Curation Task menu section",
+  "menu.section.icon.cu
```
Comment (bram-atmire) [inline resources/i18n/es.json5]:
```suggestion
  "submission.workflow.tasks.claimed.reject_help": "Si ha revisado el ítem y ha encontrado que <strong> no es </strong> adecuado para su inclusión en la colección, seleccione \"Rechazar\". Después se le pedirá que ingrese un mensaje que indique por qué el ítem no es adecuado y si el remitente debe cambiar algo y volver a enviarlo.",
```

Comment (bram-atmire) [reply resources/i18n/es.json5]:
```
The problematic piece was the escaping: \ "Rechazar \" needs to be \"Rechazar\"
```

Comment (fernandaruizm) [reply resources/i18n/es.json5]: Thanks Bram to fix the "escaping issue

**Pull request merged**